Mathlib/LinearAlgebra/FreeProduct/Basic.lean

```diff
@@ -226,10 +353,13 @@ to a unique arrow `π` from `FreeProduct R A` such that  `π ∘ ι i = maps i`.
 /--Universal property of the free product of algebras, property:
 for every `R`-algebra `B`, every family of maps `maps : (i : I) → (A i →ₐ[R] B)` lifts
 to a unique arrow `π` from `FreeProduct R A` such that  `π ∘ ι i = maps i`.-/
-theorem lift_comp_ι : (lift R A maps) ∘ₐ (ι R A i) = maps := by
+@[simp↓] theorem lift_comp_ι : (lift R A maps) ∘ₐ (ι R A i) = maps := by
```

Comment:
Why `simp↓`?

Reply:
The lhs of this definition is not in simp-normal form, but it is in, shall we say, "human-normal form" -- it's something that I expect actual users of this file to type out at some point while using it, certainly it's something I used often enough. Meanwhile, the actual simp-normal form for the lhs is an illegible mess. `@[simp↓]` marks the resulting simp lemma as a "shortcut simp" that should be applied immediately on its lhs at a high priority, before applying other simp lemmas, so that I don't need to define a separate lemma with a messy lhs just for `simp` to use.